src/pages/LoginPage/hooks.ts

```diff
@@ -80,7 +80,7 @@ const useLogin: UseLogin = () => {
                 ["redirect_uri", callbackUrl],
                 ["response_type", "code"],
                 ["state", key],
-                ["scope", ["read_api"].join(" ")],
+                ["scope", ["api"].join(" ")],
```

Comment:
umm why is that not just a string

Reply:
A few more parameters may be added to the scope. But not searching how it add